### ISSUE TITLE
Allow for rounding in tables

### ISSUE THREAD

**Issue by chrisleboa** [kaz-yos/tableone#78]
Is there any way to specify rounding in the output table from a createtableone? I find myself manually going through and deleting everything after a decimal point and would appreciate the ability to have the output be in whole numbers only

**Comment by tszberkowitz** [comment]
@chrisleboa Yes. See `?print.TableOne` for more details on the `print` method for `TableOne` objects. You probably want the `catDigits`, `contDigits`, and `pDigits` arguments but there is also the `formatOptions` argument for passing additional formatting option arguments.
